PixaPicker/PixaDataCoordinator.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  PixaDataCoordinator.swift
+//  PixaPicker
+//
+//  Created by Thomas Caraway on 11/22/19.
+//  Copyright © 2019 Thomas Caraway. All rights reserved.
+//
+
+import Foundation
+
+class PixaDataCoordinator {
+    
+    var delegate: PixaDataCoordinatorDelegate?
+    var cellImageURLs = [URL]()
```

Comment:
make `cellImageURLs` private.  

Instead, add two accessors:
`var imageCount: Int {  return cellImageURLs.count }`

Reply:
The next one is a bit more complicated. 
Add this: 
``` 
func getImageUrl(for index: Int) -> URL { 
    if shouldGetNextPage(for: index) { 
            getNextPage() 
    }
    return cellImageURLs[index]
}

Reply:
The reason you do this is so that you are only exposing what is actually needed.  The `ViewController` doesn't care about the underlying data structure whether that be an array, list, dict, anything.  It only cares about the count, and getting the next available URL, so that's all we care to expose.